src/gulpfile.js

```diff
@@ -255,6 +255,29 @@ gulp.task('build-html', ['add-revision-numbers'], function () {
   .pipe(gulp.dest(paths.build));
 });
 
+gulp.task('build-mseditor', ['template-editor', 'add-revision-numbers'], function () {
+  var sources = gulp.src([
+    paths.build + '/template-editor/relayEditor.*.js'
+  ]);
+  return gulp.src([
+    paths.app + '/template-editor/index.html'
+  ])
+  .pipe(gulpInject(sources, {
+    addRootSlash: false, // ensures proper relative paths
+    ignorePath: paths.build + '/template-editor' // ensures proper relative paths
+  }))
+  .pipe(gulp.dest(paths.build + '/template-editor'));
+});
+
+gulp.task('template-editor', function () {
+  return gulp.src([
+    paths.app + '/template-editor/adapterService.js',
+    paths.app + '/env/' + process.env.NODE_ENV + '.js'
+  ])
+  .pipe(concat('relayEditor.js'))
+  .pipe(gulp.dest(paths.tmpPrebuild + '/template-editor'));
```

Comment:
If I understand it fine, it generates a file called `relayEditor.js` in `tmpPrebuild` folder, **who** read that file?

Reply:
It is loaded by add-revision-numbers to generate revision numbers